src/tally.rs

```diff
@@ -250,10 +282,7 @@ pub async fn tally(matches: &ArgMatches) -> Result<(), AocError>
     let unimplementeds = build_days(cargo_folder, &have).await?;
 
     have.retain(|elem| !unimplementeds.contains(&elem.1));
-    for unimpl in unimplementeds
-    {
-        dont.push(unimpl);
-    }
+    dont.extend(unimplementeds);
 
```

Comment:
While you are at it 😬

We could get rid of `.cloned()` on `let silver` and `let gold` as well. If we dereference `*p1` inside the map on the line for `let silver = ...`. We can then change the type in `print_info` to expect `Vec<(&u32, usize)>`